### PR TITLE
Adds models md to Models Hub for 3.3.2 Healthcare release

### PR DESCRIPTION
**Pull request by josejuanmartinez**
## Description

Adds md models to Models Hub for 3.3.2 Healthcare release

## Motivation and Context

Healthcare is releasing 3.3.2, and several models have been uploaded to the Hub. For these new models, we need to merge the md files.

## How Has This Been Tested?

Checking models are downloadable and documentation looks ok.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Code improvements with no or little impact
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [CONTRIBUTING](http://nlp.johnsnowlabs.com/contribute.html) page.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/_posts/HashamUlHaq/2021-11-05-ner_covid_trials_en.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
+---
+layout: model
+title: Extract entities in covid trials
+author: John Snow Labs
+name: ner_covid_trials
+date: 2021-11-05
+tags: [ner, en, clinical, licensed]
+task: Named Entity Recognition
+language: en
+edition: Spark NLP for Healthcare 3.2.3
```
Comment (josejuanmartinez) [inline docs/_posts/HashamUlHaq/2021-11-05-ner_covid_trials_en.md:10]: Shouldn't this be 3.3.2, or at least 3.3.1?

File: docs/_posts/HashamUlHaq/2021-11-05-ner_covid_trials_en.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+|Output Labels:|[ner]|
+|Language:|en|
+
+## Data Source
+
+This model is trained on data sampled from clinicaltrials.gov - covid trials, and annotated in-house.
+
+## Benchmarking
+
+```bash
```
Comment (josejuanmartinez) [inline docs/_posts/HashamUlHaq/2021-11-05-ner_covid_trials_en.md:111]: This is bad formatted. We need to have columns aligned.

File: docs/_posts/galiph/2021-11-04-sbiobertresolve_rxnorm_augmented_en.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
+---
+layout: model
+title: Sentence Entity Resolver for RxNorm (sbiobert_base_cased_mli embeddings)
+author: John Snow Labs
+name: sbiobertresolve_rxnorm_augmented
+date: 2021-11-04
+tags: [rxnorm, licensed, en, clinical, entity_resolution]
+task: Entity Resolution
+language: en
+edition: Spark NLP for Healthcare 3.3.1
```
Comment (josejuanmartinez) [inline docs/_posts/galiph/2021-11-04-sbiobertresolve_rxnorm_augmented_en.md:10]: 3.3.1 or 3.3.2?

File: docs/_posts/murat-gunay/2021-11-03-bert_token_classifier_ner_bionlp_en.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+...
+tokenClassifier = BertForTokenClassification.pretrained("bert_token_classifier_ner_bionlp", "en", "clinical/models")
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-03-bert_token_classifier_ner_bionlp_en.md:39]: We need the `\` at the end of each line or the code won't execute

File: docs/_posts/murat-gunay/2021-11-03-bert_token_classifier_ner_cellular_en.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+...
+tokenClassifier = BertForTokenClassification.pretrained("bert_token_classifier_ner_cellular", "en", "clinical/models")
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-03-bert_token_classifier_ner_cellular_en.md:39]: We need the `\` or the code won't execute

File: docs/_posts/murat-gunay/2021-11-04-ner_chemd_clinical_en.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+...
+word_embeddings = WordEmbeddingsModel.pretrained("embeddings_clinical", "en", "clinical/models")\ 
+      .setInputCols(["sentence", "token"])\ 
+      .setOutputCol("embeddings")
+
+chemd_ner = MedicalNerModel.pretrained('ner_chemd', 'en', 'clinical/models') \
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-04-ner_chemd_clinical_en.md:51]: `ner_chemd` can't be found. I think it should be `ner_chemd_clinical`

File: docs/_posts/murat-gunay/2021-11-04-ner_chemd_clinical_en.md [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+
+empty_data = spark.createDataFrame([[""]]).toDF("text")
+
+chemd_model = nlpPipeline.fit(empty_data)
+
+results = model.transform(spark.createDataFrame(pd.DataFrame({"text": ["""Isolation, Structure Elucidation, and Iron-Binding Properties of Lystabactins, Siderophores Isolated from a Marine Pseudoalteromonas sp. The marine bacterium Pseudoalteromonas sp. S2B, isolated from the Gulf of Mexico after the Deepwater Horizon oil spill, was found to produce lystabactins A, B, and C (1-3), three new siderophores. The structures were elucidated through mass spectrometry, amino acid analysis, and NMR. The lystabactins are composed of serine (Ser), asparagine (Asn), two formylated/hydroxylated ornithines (FOHOrn), dihydroxy benzoic acid (Dhb), and a very unusual nonproteinogenic amino acid, 4,8-diamino-3-hydroxyoctanoic acid (LySta). The iron-binding properties of the compounds were investigated through a spectrophotometric competition."""]})))
+```
+```scala
+...
+val word_embeddings = WordEmbeddingsModel.pretrained("embeddings_clinical", "en", "clinical/models")
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-04-ner_chemd_clinical_en.md:69]: I think these `\` are needed

File: docs/_posts/murat-gunay/2021-11-04-ner_chemd_clinical_en.md [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+val chemd_ner = MedicalNerModel.pretrained("ner_chemd", "en", "clinical/models") 
+      .setInputCols(Array("sentence", "token", "embeddings")) 
+      .setOutputCol("ner")
+
+val ner_converter = NerConverter()
+      .setInputCols(Array("sentence", "token", "ner"))
+      .setOutputCol("ner_chunk")
+
+val nlpPipeline = new Pipeline().setStages(Array(document_assembler, sentence_detector, tokenizer, word_embeddings, chemd_ner, ner_converter))
+val model = nlpPipeline.fit(spark.createDataFrame([[""]]).toDF("text"))
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-04-ner_chemd_clinical_en.md:82]: I think this line is for Python

File: docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_antisemitism_en.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+spark_version: 2.4
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model is imported from `Hugging Face-models and it was trained on 4K tweets, where ~50% were labeled as antisemitic. The model identifies if the text is antisemitic or not.
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_antisemitism_en.md:20]: Missing a close quote after Hugging Face-models

File: docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_antisemitism_en.md [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+      .setInputCols(['token', 'document']) \
+      .setOutputCol('class')
+
+pipeline = Pipeline(stages=[document_assembler, tokenizer, sequenceClassifier])
+
+example = spark.createDataFrame([["The Jews have too much power!"]]).toDF("text")
+result = pipeline.fit(example).transform(example)
+```
+```scala
+val document_assembler = DocumentAssembler()
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_antisemitism_en.md:59]: `\` are these needed?

File: docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_hatexplain_en.md [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+      .setCaseSensitive(True) \
+      .setMaxSentenceLength(512)
+
+pipeline = Pipeline(stages=[document_assembler, tokenizer, sequenceClassifier])
+
+example = spark.createDataFrame([['I love you very much!']]).toDF("text")
+result = pipeline.fit(example).transform(example)
+```
+```scala
+val document_assembler = DocumentAssembler()
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_hatexplain_en.md:69]: `\`  are these needed?

File: docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_trec_coarse_en.md [hunk 12]
```diff
@@ -0,0 +1,103 @@
...
+      .setCaseSensitive(True) \
+      .setMaxSentenceLength(512)
+
+pipeline = Pipeline(stages=[document_assembler, tokenizer, sequenceClassifier])
+
+example = spark.createDataFrame([['Germany is the largest country in Europe economically.']]).toDF("text")
+result = pipeline.fit(example).transform(example)
+```
+```scala
+val document_assembler = DocumentAssembler()
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_trec_coarse_en.md:59]: `\` are these needed?

File: docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_banking77_en.md [hunk 13]
```diff
@@ -0,0 +1,103 @@
...
+      .setCaseSensitive(True) \
+      .setMaxSentenceLength(512)
+
+pipeline = Pipeline(stages=[document_assembler, tokenizer, sequenceClassifier])
+
+example = spark.createDataFrame([['I am still waiting on my card?']]).toDF("text")
+result = pipeline.fit(example).transform(example)
+```
+```scala
+val document_assembler = DocumentAssembler()
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_banking77_en.md:59]: `\` are these needed?

File: docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_age_news_en.md [hunk 14]
```diff
@@ -0,0 +1,103 @@
...
+      .setCaseSensitive(True) \
+      .setMaxSentenceLength(512)
+
+pipeline = Pipeline(stages=[document_assembler, tokenizer, sequenceClassifier])
+
+example = spark.createDataFrame([['Microsoft has taken its first step into the metaverse.']]).toDF("text")
+result = pipeline.fit(example).transform(example)
+```
+```scala
+val document_assembler = DocumentAssembler()
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_age_news_en.md:59]: `\` are these needed?

File: docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_age_news_en.md [hunk 15]
```diff
@@ -0,0 +1,103 @@
...
+|License:|Open Source|
+|Edition:|Official|
+|Input Labels:|[token, sentence]|
+|Output Labels:|[label]|
+|Language:|en|
+|Case sensitive:|true|
+
+## Data Source
+
+[https://huggingface.co/mrm8488/bert-mini-finetuned-age_news-classification](https://huggingface.co/mrm8488/bert-mini-finetuned-age_news-classification)
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_age_news_en.md:103]: There is a metric here we can include as benchmark: 
image

File: docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_trec_coarse_en.md [hunk 16]
```diff
@@ -0,0 +1,103 @@
...
+|License:|Open Source|
+|Edition:|Official|
+|Input Labels:|[token, sentence]|
+|Output Labels:|[label]|
+|Language:|en|
+|Case sensitive:|true|
+
+## Data Source
+
+[https://huggingface.co/aychang/bert-base-cased-trec-coarse](https://huggingface.co/aychang/bert-base-cased-trec-coarse)
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_trec_coarse_en.md:103]: There are metrics available: 
image

File: docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_hatexplain_en.md [hunk 17]
```diff
@@ -0,0 +1,113 @@
...
+|License:|Open Source|
+|Edition:|Official|
+|Input Labels:|[token, sentence]|
+|Output Labels:|[label]|
+|Language:|en|
+|Case sensitive:|true|
+
+## Data Source
+
+[https://huggingface.co/Hate-speech-CNERG/bert-base-uncased-hatexplain](https://huggingface.co/Hate-speech-CNERG/bert-base-uncased-hatexplain)
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_hatexplain_en.md:113]: There are metrics here: 
image

Use AUCROC BERT-HateXplain [LIME]

File: docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_banking77_en.md [hunk 18]
```diff
@@ -0,0 +1,103 @@
...
+|License:|Open Source|
+|Edition:|Official|
+|Input Labels:|[token, sentence]|
+|Output Labels:|[label]|
+|Language:|en|
+|Case sensitive:|true|
+
+## Data Source
+
+[https://huggingface.co/philschmid/BERT-Banking77](https://huggingface.co/philschmid/BERT-Banking77)
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_banking77_en.md:103]: There is a benchmark we can use: 
image

File: docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_sms_spam_en.md [hunk 19]
```diff
@@ -0,0 +1,106 @@
...
+      .setCaseSensitive(True) \
+      .setMaxSentenceLength(512)
+
+pipeline = Pipeline(stages=[document_assembler, tokenizer, sequenceClassifier])
+
+example = spark.createDataFrame([['Camera - You are awarded a SiPix Digital Camera! call 09061221066 from landline. Delivery within 28 days.']]).toDF("text")
+result = pipeline.fit(example).transform(example)
+```
+```scala
+val document_assembler = DocumentAssembler()
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_sms_spam_en.md:62]: Are these needed: `\` ?

File: docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_sms_spam_en.md [hunk 20]
```diff
@@ -0,0 +1,106 @@
...
+|License:|Open Source|
+|Edition:|Official|
+|Input Labels:|[token, sentence]|
+|Output Labels:|[label]|
+|Language:|en|
+|Case sensitive:|true|
+
+## Data Source
+
+[https://huggingface.co/mrm8488/bert-tiny-finetuned-sms-spam-detection](https://huggingface.co/mrm8488/bert-tiny-finetuned-sms-spam-detection)
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_sms_spam_en.md:106]: We have metrics in HF: 
image

File: docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_song_lyrics_en.md [hunk 21]
```diff
@@ -0,0 +1,103 @@
...
+sequenceClassifier = BertForSequenceClassification \
+      .pretrained('bert_sequence_classifier_song_lyrics', 'en') \
+      .setInputCols(['token', 'document']) \
+      .setOutputCol('class') \
+      .setCaseSensitive(True) \
+      .setMaxSentenceLength(512)
+
+pipeline = Pipeline(stages=[document_assembler, tokenizer, sequenceClassifier])
+
+example = spark.createDataFrame([['Because you need me Every single day Trying to find me But you don't know why Trying to find me again But you don't know how Trying to find me again Every single day']]).toDF("text")
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_song_lyrics_en.md:55]: This won't work, conflicting `'` . Start and finish with `"`

File: docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_song_lyrics_en.md [hunk 22]
```diff
@@ -0,0 +1,103 @@
...
+result = pipeline.fit(example).transform(example)
+```
+```scala
+val document_assembler = DocumentAssembler()
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_song_lyrics_en.md:59]: Are `\` needed?

File: docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_song_lyrics_en.md [hunk 23]
```diff
@@ -0,0 +1,103 @@
...
+|License:|Open Source|
+|Edition:|Official|
+|Input Labels:|[token, sentence]|
+|Output Labels:|[label]|
+|Language:|en|
+|Case sensitive:|true|
+
+## Data Source
+
+[https://huggingface.co/juliensimon/autonlp-song-lyrics-18753417](https://huggingface.co/juliensimon/autonlp-song-lyrics-18753417)
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_song_lyrics_en.md:103]: We have metrics: 
image

File: docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_antisemitism_en.md (hunk 10 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_antisemitism_en.md:59]: @josejuanmartinez `\` is not needed in scala script.

File: docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_hatexplain_en.md (hunk 11 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_hatexplain_en.md:69]: @josejuanmartinez `\` is not needed in scala script.

File: docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_trec_coarse_en.md (hunk 12 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_trec_coarse_en.md:59]: @josejuanmartinez `\` is not needed in scala script.

File: docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_banking77_en.md (hunk 13 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_banking77_en.md:59]: @josejuanmartinez `\` is not needed in scala script.

File: docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_age_news_en.md (hunk 14 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_age_news_en.md:59]: @josejuanmartinez `\` is not needed in scala script.

File: docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_sms_spam_en.md (hunk 19 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_sms_spam_en.md:62]: @josejuanmartinez `\` is not needed in scala script.

File: docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_song_lyrics_en.md (hunk 22 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-11-07-bert_sequence_classifier_song_lyrics_en.md:59]: @josejuanmartinez `\` is not needed in scala script.

File: docs/_posts/HashamUlHaq/2021-11-05-ner_covid_trials_en.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/HashamUlHaq/2021-11-05-ner_covid_trials_en.md:10]: it should be 3.3.2

File: docs/_posts/murat-gunay/2021-11-04-ner_chemd_clinical_en.md (hunk 6 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-11-04-ner_chemd_clinical_en.md:51]: yes `ner_chemd_clinical` works

File: docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_antisemitism_en.md (hunk 9 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-11-06-bert_sequence_classifier_antisemitism_en.md:20]: yes it should be like this -->  `Hugging Face-models`

**Review by josejuanmartinez — COMMENTED** [review]
Now it seems much better. Thank you @gadde5300 !

**Pull request merged**